lessons/python_basics/IntroPython.ipynb

```diff
@@ -0,0 +1,1303 @@
+     "output_type": "stream",
+     "text": [
+      "[ 0  1  2  3  4  5  6  7  8  9 10 11 12 13 14]\n",
+      "[0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14]\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(array)\n",
+    "print(lst)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 53,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<type 'numpy.ndarray'>\n",
+      "<type 'list'>\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(type(array))\n",
+    "print(type(lst))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 55,
+   "metadata": {
+    "collapsed": false,
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[ 0  2  4  6  8 10 12 14 16 18 20 22 24 26 28]\n",
+      "[0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14]\n"
+     ]
+    }
+   ],
+   "source": [
+    "# numpy arrays allow for vectorized calculations\n",
+    "print(array*2)\n",
+    "print(lst*2)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 57,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[[ 0  1  2]\n",
+      " [ 3  4  5]\n",
+      " [ 6  7  8]\n",
+      " [ 9 10 11]\n",
+      " [12 13 14]]\n"
+     ]
+    }
+   ],
+   "source": [
+    "array = array.reshape([5,3])\n",
+    "print(array)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 59,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([  1.,   4.,   7.,  10.,  13.])"
+      ]
+     },
+     "execution_count": 59,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# we can get the mean over all rows (using axis=1)\n",
+    "array.mean(axis=1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 60,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([12, 13, 14])"
+      ]
+     },
+     "execution_count": 60,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# max value in each column\n",
+    "array.max(axis=0)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 61,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import pandas as pd"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 62,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# this will read in a csv file into a pandas DataFrame\n",
+    "# this csv has data of country spending on healthcare\n",
+    "data = pd.read_csv('data/health.csv', header=0, index_col=0)"
```

Comment:
Hey, I forgot I was using the wrong environment, I'll update with a Py3 terminal and also include the csv, the last part was mostly a demo rather than a code-along so I didn't provide it ahead of time.